sharding/test/compression_verifier_test.go

```diff
@@ -67,6 +67,13 @@ func (t *CompressionVerifierTestSuite) InsertCompressedRowInDb(id int, schema, d
 	t.Require().Nil(err)
 }
 
+func (t *CompressionVerifierTestSuite) replaceCompressionVerifier(tableCompressions ghostferry.TableColumnCompressionConfig) {
+	var err error
+	iterativeVerifier := t.Ferry.Ferry.Verifier.(*ghostferry.IterativeVerifier)
+	iterativeVerifier.CompressionVerifier, err = ghostferry.NewCompressionVerifier(tableCompressions)
+	t.Require().Nil(err)
+}
```

Comment:
This is..really ugly. Can you do any better with some more refactoring?

Reply:
I just tried and I can't do this without major rework of this test. At that point, it would be nice to be able to extract these tests out to the ruby tests, where we can more easily handle configuration changes.

Between this and the issue with table loading in the Go integration tests, we might want to think about refactoring all the tests into the ruby tests. However, I'm not sure if we'll have time to do this for now.